src/MovieDetailContainer.js

```diff
@@ -1,18 +1,64 @@
-import React from 'react'
+import { React, Component } from 'react'
 import MovieDetail from './MovieDetail'
 import './MovieDetailContainer.css'
 
-const MovieDetailContainer = ({ selectedDetails }) => {
-  return (
-    <section className='movie-detail-container'>
-      <div className='title'>
-        <h1>Selected movie</h1>
-      </div>
-      <div className='movie-detail'>
-        <MovieDetail details={selectedDetails}/>
-      </div>
-    </section>
-  )
-}
+class MovieDetailContainer extends Component {
```

Comment:
Thank you for Refactoring this function to class component and moving our recent state to this component!